src/index.ts

```diff
@@ -136,6 +138,7 @@ export const swagger =
                     openapi: '3.0.3',
                     ...{
                         ...documentation,
+                        tags: (documentation?.tags as {name: string; description: string}[])?.filter((tag) => !excludeTags?.includes(tag?.name)),
```

Comment:
I think we can safely use the OpenAPI type and make the filter 

```
tags: (documentation.tags?.filter((tag) => !excludeTags?.includes(tag?.name)))
```

Reply:
@marclave sure, I removed the type